crates/matrix-sdk-ui/src/timeline/event_item/content/msg_like.rs

```diff
@@ -36,17 +36,31 @@ pub enum MsgLikeKind {
     UnableToDecrypt(EncryptedMessage),
 }
 
+#[derive(Clone, Debug)]
+pub struct ThreadSummary {
+    pub latest_event_details: TimelineDetails<Box<ThreadSummaryLatestEvent>>,
+}
+
+#[derive(Clone, Debug)]
+pub struct ThreadSummaryLatestEvent {
+    pub content: TimelineItemContent,
+    pub sender: OwnedUserId,
+    pub sender_profile: TimelineDetails<Profile>,
+}
+
 /// A special kind of [`super::TimelineItemContent`] that groups together
 /// different room message types with their respective reactions and thread
 /// information.
 #[derive(Clone, Debug)]
 pub struct MsgLikeContent {
     pub kind: MsgLikeKind,
     pub reactions: ReactionsByKeyBySender,
-    /// Event ID of the thread root, if this is a threaded message.
-    pub thread_root: Option<OwnedEventId>,
     /// The event this message is replying to, if any.
     pub in_reply_to: Option<InReplyToDetails>,
+    /// Event ID of the thread root, if this is a threaded message.
```

Comment:
Let's try to make it super clear the difference between thread root and messages-in-thread. "Threaded message" could be the root, in my head.

```suggestion
    /// Event ID of the thread root, if this is a message in a thread.
```